synapse/third_party_rules/access_rules.py

```diff
@@ -742,11 +753,14 @@ def _is_power_level_content_allowed(
             # which are compliant
 
             # If invite requirements are <PL50
-            if content.get("invite", 50) < 50:
+            if content.get("invite", default_power_levels["invite"]) < 50:
```

Comment:
you have to replace the threshold too
"if content.get("invite", default_power_levels["invite"]) < default_power_levels["invite"]:"

same for `default_power_levels["invite"]` below

Reply:
Ah right, of course. Have fixed.